init.coffee

```diff
@@ -13,11 +13,30 @@ exports.configure = [
     message: 'Do you need netlify hosting?'
   },
   {
+    type: 'list'
+    name: 'css'
+    message: 'What would you like to use as your CSS Preprocessor?'
+    choices: ['postcss', 'stylus']
```

Comment:
my vote is we use stylus and postcss by default, actually. since that's how we practically use it now